app/controllers/visit_requests_controller.rb

```diff
@@ -32,4 +32,8 @@ def event
   def visit_request
     @visit_request ||= VisitRequest.find(params[:id])
   end
+
+  def user_verification_status
+    current_user.verified? ? :success_for_verified : :success_for_newbies
```

Comment:
Actually I really don't get how this will improve this code.
Can you give a pseudo-code example?